external/curl/curl_config.h

```diff
@@ -862,8 +862,10 @@
 /* Use TLS-SRP authentication */
 #define USE_TLS_SRP 1
 
+#ifndef CONFIG_ENABLE_CURL
```

Comment:
Let's use ```CONFIG_NET_LWIP``` instead of ```CONFIG_ENABLE_CURL```.
Logically it is not the approprietary reason to block unix socket. LWIP does not support. NOT curl does not support.

Reply:
To use CONFIG_XXX definition, you MUST include <tinyara/config.h>

Reply:
Pilseob said, LWIP and socket are different layer so that making dependacy with LWIP config is incorrect for this.
His recommandation is __TIZENRT__  like another extern module.
```
CFLAGS += -D__TizenRT__
```

Reply:
Okey